core/src/test/java/org/infinispan/stats/SingleStatsTest.java

```diff
@@ -0,0 +1,115 @@
+package org.infinispan.stats;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+
+import org.infinispan.Cache;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.configuration.cache.StorageType;
+import org.infinispan.eviction.EvictionType;
+import org.infinispan.persistence.dummy.DummyInMemoryStoreConfigurationBuilder;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "stats.SingleStatsTest")
+public class SingleStatsTest extends MultipleCacheManagersTest {
+
+   protected final int EVICTION_MAX_ENTRIES = 3;
+   protected final int TOTAL_ENTRIES = 5;
+   protected StorageType storageType;
+   protected Cache cache;
+   protected Stats stats;
+
+   @Factory
+   public Object[] factory() {
+      return new Object[]{
+            //new SingleStatsTest().withStorage(StorageType.OBJECT),
```

Comment:
Do we want to reenable this?

Reply:
yes, thanks! Added also `BINARY` for sure.